src/libraries/System.Private.CoreLib/src/System/IO/UnmanagedMemoryAccessor.cs

```diff
@@ -1,26 +1,20 @@
 // Licensed to the .NET Foundation under one or more agreements.
 // The .NET Foundation licenses this file to you under the MIT license.
 
-/*============================================================
-**
-**
-**
-**
-** Purpose: Provides a fast, AV free, cross-language way of
-**          accessing unmanaged memory in a random fashion.
-**
-**
-===========================================================*/
-
 using System.Diagnostics;
 using System.Runtime.InteropServices;
 using System.Runtime.CompilerServices;
 
 namespace System.IO
 {
+    /// <summary>
+    /// Provides random access to unmanaged blocks of memory from managed code.
+    /// </summary>
+    /// <remarks>
     /// Perf notes: ReadXXX, WriteXXX (for basic types) acquire and release the
     /// SafeBuffer pointer rather than relying on generic Read(T) from SafeBuffer because
     /// this gives better throughput; benchmarks showed about 12-15% better.
+    /// </remarks>
```

Comment:
Seem like maintainer comments. Need to be double slash:

```suggestion
    // Perf notes: ReadXXX, WriteXXX (for basic types) acquire and release the
    // SafeBuffer pointer rather than relying on generic Read(T) from SafeBuffer because
    // this gives better throughput; benchmarks showed about 12-15% better.
```